compiler/testSuite/01-function/call13-e.bal

```diff
@@ -1,6 +1,6 @@
 // @productions return-stmt local-var-decl-stmt int-literal
 public function main() {
-    int x = 1;
+    _ = 1;
```

Comment:
This is not an OK change!!!!! Fix like this:

```
int x = 1;
_ = x;
```

Reply:
@jclark, since we are testing `unreachable code` error in this test case, can you explain to me the reason for the above particular change? 
What I thought was, since we are testing `unreachable code`, it does not matter the statements we have before the return-stmt.

Reply:
You are changing what the test is testing.  One of the things that the original test is checking is that `1` can be assigned to a variable of type `int`. With your change, that is no longer being tested. You should only ever be adding lines that are assignments to `_`.

Reply:
Thank you, I understood.
Fixed

Reply:
Five exclamation marks, the sure sign of an insane mind. ― Sir Terry Pratchett